src/magic_folder/test/test_snapshot.py

```diff
@@ -99,10 +106,22 @@ def setUp(self):
         self.alice = create_local_author("alice")
         self.stash_dir = mktemp()
         os.mkdir(self.stash_dir)
+
+        # create a magicfolder db
+        self.tempdb=abspath_expanduser_unicode(unicode(mktemp()))
+        os.mkdir(self.tempdb)
```

Comment:
We should avoid using the `unicode` keyword which is python2-only. I think this can just be "`self.tempdb = mktemp()`" though like `stash_dir` above?